3p/integration.js

```diff
@@ -78,6 +78,7 @@ import {adyoulike} from '#ads/vendors/adyoulike';
 import {affiliateb} from '#ads/vendors/affiliateb';
 import {aja} from '#ads/vendors/aja';
 import {amoad} from '#ads/vendors/amoad';
+import {amplified} from '#ads/vendors/amplified';
```

Comment:
I think we no longer need changes in this file. This likely changed when this PR was out for review. @powerivq to confirm.

Reply:
Yes. You are correct.